### PR TITLE
Prepare to publish build_modules

### PR DESCRIPTION
**Pull request by natebosch**

### CODE REVIEW

File: build_modules/mono_pkg.yaml [hunk 1]
```diff
@@ -12,7 +12,8 @@ stages:
   - unit_test:
     # Run the script directly - running from snapshot has issues for packages
     # that are depended on by build_runner itself.
-    - command: dart $(pub run build_runner generate-build-script) test --delete-conflicting-outputs-- -P presubmit
+    #- command: dart $(pub run build_runner generate-build-script) test --delete-conflicting-outputs-- -P presubmit
```
Comment (kevmoo) [inline build_modules/mono_pkg.yaml]: Maybe add a note here about why we're commenting this out?

File: build_modules/pubspec.yaml [hunk 2]
```diff
@@ -26,10 +26,9 @@ dependencies:
 
 dev_dependencies:
   build_runner: ^1.0.0
-  build_test: ^0.10.0
-  build_vm_compilers: ">=0.1.0 <2.0.0"
+  #build_test: ^0.10.0
```
Comment (kevmoo) [inline build_modules/pubspec.yaml]: note about commented-out things? Or just remove em?

Comment (natebosch) [reply build_modules/pubspec.yaml]: Decided to restore them and bump the min SDK instead.

I realized there isn't any value in supporting an older SDK than is supported by the only 2 packages that allow this version of `build_modules` 😝 

Technically `build_node_compilers` could end up supporting this version and older SDK than `build_vm_compilers`, but I think that's unlikely...

**Comment by kevmoo** [comment]
no worries

**Pull request merged**